frontend/packages/shared/src/utils/featureToggleUtils.ts

```diff
@@ -1,17 +1,15 @@
-import {
-  typedLocalStorage,
-  typedSessionStorage,
-} from "app-shared/utils/webStorage";
+import { typedLocalStorage, typedSessionStorage } from 'app-shared/utils/webStorage';
 
-const featureFlagKey = "featureFlags";
-const persistFeatureKey = "persistFeatureFlag";
+const featureFlagKey = 'featureFlags';
+const persistFeatureKey = 'persistFeatureFlag';
 
 // All the features that you want to be toggle on/off should be added here. To ensure that we type check the feature name.
 export type SupportedFeatureFlags =
-  | "componentConfigBeta"
-  | "expressions"
-  | "policyEditor"
-  | "processEditor";
+  | 'componentConfigBeta'
+  | 'expressions'
+  | 'policyEditor'
+  | 'processEditor'
+  | 'configureLayoutSet';
```

Comment:
I have just added this feature-flag, the rest is prettier formatting done by the pre-commit.